src/Tasks/PageIterator.php

```diff
@@ -67,6 +71,7 @@ public function setPauseIndex(int $pauseIndex): void
     }
 
     /**
+     * @param callable $callback The callback function to apply on every entity.
```

Comment:
```suggestion
     * @param callable(mixed): bool $callback The callback function to apply on every entity. Pauses iteration if false is returned
```
From https://phpstan.org/writing-php-code/phpdoc-types#callables